docs/compute-entrypoint/htcondor-ce-overview.md

```diff
@@ -13,10 +13,10 @@ Before continuing with the overview, make sure that you are familiar with the fo
 -   Pilot jobs, frontends, and factories (i.e., [GlideinWMS](http://glideinwms.fnal.gov/doc.prd/index.html),
     AutoPyFactory)
 
-What is a Compute Element?
+What is a Compute Entrypoint?
 --------------------------
 
-An OSG Compute Element (CE) is the entry point for the OSG to your local resources: a layer of software that you install
+An OSG Compute Entrypoint (CE) is the entry point for the OSG to your local resources: a layer of software that you install
 on a machine that can submit jobs into your local batch system.
```

Comment:
```suggestion
An OSG Compute Entrypoint (CE) is the door for remote organizations to submit requests to temporarily allocate local
compute resources.
```

I didn't love the doubling up of "Entrypoint" and "entry point"

Reply:
Knock on the OSG Door of Opportunity (DO) !